controllers/controllers/resource/fetcher.go

```diff
@@ -41,35 +41,35 @@ type ResourceFetcher interface {
 	OIDCConfig(ctx context.Context, ref *anywherev1.Ref, namespace string) (*anywherev1.OIDCConfig, error)
 }
 
-type capiResourceFetcher struct {
+type CapiResourceFetcher struct {
 	client client.Reader
 	log    logr.Logger
 }
 
-func NewCAPIResourceFetcher(client client.Reader, Log logr.Logger) *capiResourceFetcher {
-	return &capiResourceFetcher{
+func NewCAPIResourceFetcher(client client.Reader, Log logr.Logger) *CapiResourceFetcher {
```

Comment:
are we still using the old controller fetcher.go? somehow i thought we are moving away from it

Reply:
not anymore. The code is now using Client.Get